src/lib.rs

```diff
@@ -70,7 +63,7 @@ impl FilePerThreadLogger {
 
 impl log::Log for FilePerThreadLogger {
     fn enabled(&self, metadata: &Metadata) -> bool {
-        enabled() && self.filter.enabled(metadata)
```

Comment:
In the original code, this enabled() check was meant to be very fast, to minimize the overhead when logging isn't initialized. Do I understand correctly that that's not important now, because if logging isn't initialized, this function won't even be called?

Reply:
Yes, if logging hasn't been enabled, the logging implementation just ignores all the messages. The `enabled()` check only happens if we've set the logger, thus if we've set the env variable.